packages/bsky/service/api.js

```diff
@@ -65,12 +66,22 @@ const main = async () => {
   })
   const viewMaintainer = new ViewMaintainer(migrateDb)
   const viewMaintainerRunning = viewMaintainer.run()
+  
+  const periodicModerationActionReversal = new PeriodicModerationActionReversal(
+    db,
```

Comment:
should we just pass `pds` here and grab the db instance from pds inside the `PeriodicModerationActionReversal` class?